packages/slice-machine/src/domain/fields.ts

```diff
@@ -0,0 +1,202 @@
+import { IconName } from "@prismicio/editor-ui";
+import {
+  BooleanFieldType,
+  ColorFieldType,
+  DateFieldType,
+  EmbedFieldType,
+  FieldType,
+  GeoPointFieldType,
+  GroupFieldType,
+  ImageFieldType,
+  LinkFieldType,
+  NestableFieldTypes,
+  NumberFieldType,
+  RichTextFieldType,
+  SelectFieldType,
+  TextFieldType,
+  TimestampFieldType,
+  UIDFieldType,
```

Comment:
💡 In the past we've discussed a lot with other folks about removing the usage of these constants and use string literals instead. This is because for TypeScript they're the same, but for the developer is fewer things to maintain and less indirection.

Reply:
I wanted to be the good dev here haha, I started without, If we believe we can slowly stop using them I'm up for it. Just need to be careful with it in PRs.

Reply:
Done